uxarray/grid/validation.py

```diff
@@ -111,37 +112,30 @@ def _check_normalization(grid):
         return grid._normalized
 
     if "node_x" in grid._ds:
-        if not (
-            np.isclose(
-                (grid.node_x**2 + grid.node_y**2 + grid.node_z**2),
-                1.0,
-                atol=ERROR_TOLERANCE,
-            )
-        ).all():
+        if not _are_coordinates_normalized(
+            grid.node_x.values, grid.node_y.values, grid.node_z.values
```

Comment:
These Numpy conversions are now needed since `_are_coordinates_normalized ` is decorate with Numba, right? This one can be a good sample case to start exploring `apply_ufunc` to avoid Numpy conversions for Xarray data structures soon. 

Looks like we don't have an issue about further avoiding whole-array computations even in Numba decorated workflows, I will open one if so

Reply:
Created #1213